app/javascript/dashboard/components/ChatList.vue

```diff
@@ -295,4 +349,24 @@ export default {
     flex-basis: 46rem;
   }
 }
+.filter--actions {
+  display: flex;
+  align-items: center;
+  .btn-filter {
+    margin: 0 var(--space-normal);
+    cursor: pointer;
+    i {
+      font-size: 2rem;
+    }
+  }
+  .btn-clear-filters {
+    color: tomato;
+    cursor: pointer;
+  }
+}
+
+.filter__applied {
+  padding: var(--space-slab) 0 !important;
+  border-bottom: 1px solid $color-border;
```

Comment:
Please use the CSS variable. https://www.chatwoot.com/docs/handbook/engineering/frontend-guidelines#use-css-variables-instead-of-scss-variables